frontend/src/app/layout/menu/menu.component.html

```diff
@@ -0,0 +1,3 @@
+<div *ngIf="authService.isLoggedIn()">
+  <button (click)="authService.logout()">Log out</button>
+</div>
```

Comment:
It would be nice to have a `Log in` button as an `else` case.

Reply:
Done